src/thunderbots/software/network_input/networking/network_client.cpp

```diff
@@ -23,7 +23,7 @@ NetworkClient::NetworkClient(ros::NodeHandle& node_handle)
     // Set up our connection over udp to receive vision packets
     try
     {
-        std::string vision_multicast_address;
+        std::string vision_multicast_address = "224.5.23.2";
```

Comment:
Is this address somehow meaningful? Is it the EDC address, or maybe the one at competition? We should give it a name, like `RANDOM_MULTICAST_ADDRESS` or `UBC_EDC_BAY_MULTICAST_ADDRESS`

Reply:
It's the default address used by SSL Vision, and by extension at competition and on our field. Something like DEFAULT_MULTICAST_ADDRESS would be fine with me

Reply:
Ah, maybe `ROBOCUP_SSL_DEFAULT_MULTICAST_ADDRESS`?

Reply:
Changed to `SSL_VISION_DEFAULT_MULTICAST_ADDRESS`